src/eth1spec/evm/__init__.py

```diff
@@ -45,7 +45,7 @@ class Evm:
 
     pc: Uint
     stack: List[U256]
-    memory: bytes
+    memory: bytearray
```

Comment:
Would `Bytes` be even better?

Reply:
`Bytes` is `bytes` though

Reply:
For type checking, `bytes` is a superclass of `bytearray` IIUC

Reply:
o